core/math/convex_hull.h

```diff
@@ -49,7 +49,7 @@ subject to the following restrictions:
 #include "core/templates/vector.h"
 
 /// Convex hull implementation based on Preparata and Hong
-/// See http://code.google.com/p/bullet/issues/detail?id=275
+/// See https://code.google.com/p/bullet/issues/detail?id=275
```

Comment:
This was bogus, the https link gives error 502. The http link worked.

Reply:
HTTP link redirects to https://code.google.com/archive/p/bullet/issues/275, so that's what we should've used.

Reply:
My bad, I thought I tested them all but I guess I missed this one.